_projects/2023/100510783/100510783.Rmd

```diff
@@ -0,0 +1,277 @@
+---
+title: "With whom do we spend our time in 2020?"
+description: |
+  A short description of the post.
+categories: "2023"
+author: Eric Hausken-Brates
+date: "`r Sys.Date()`" 
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+
+library(tidyverse)
+```
+
+## Original chart 
+```{r echo=FALSE, fig.cap="Source: FlowingData", layout = 'l-body-outset', fig.align='center', out.extra="class=external"}
+knitr::include_graphics("Screenshot.png")
```

Comment:
Apart from the graph itself, it would be nice to have some context. What's the source? Why this graph?